README.md

```diff
@@ -1 +1,187 @@
+
+```java
+onfido.applicant.create(<REQUEST_BODY_HERE>);
+```
+
+All request bodies can be created using a builder pattern on the `Request` subclass of the desired resource. For example in the request above an `Applicant.Request` object is needed as the body, this would look something like:
+
+```java
+Applicant.request().firstName("First").lastName("Last");
+```
+
+The above will return an Applicant.Request object that can be provided to the `create()` method as the request body, so a full call to to the API will look something like:
+
+```java
+Applicant newApplicant = onfido.applicant.create(
+    Applicant.request().firstName("First").lastName("Last")
+);
+```
+
+The above method will return the corresponding resource object, as most methods will. In this particular case an `Applicant` object will return with all information present for the newly created applicant. To access the information call the getter of the desired property on the object, for example:
+
+```java
+newApplicant.getFirstName();
+```
+
+## Resources
+
+All resources share the same interface when making API calls. Use `.create` to create a resource, `.find` to find one, and `.list` to fetch all resources.
+
+#### Applicants
+
+Applicants are the object upon which Onfido checks are performed.
+
+```java
+onfido.applicant.create(requestBody)                    # => Creates an applicant
+onfido.applicant.update("applicant_id", requestBody)    # => Updates an applicant
+onfido.applicant.destroy("applicant_id")                # => Schedule an applicant for deletion
+onfido.applicant.restore("applicant_id")                # => Restore an applicant scheduled for deletion
+onfido.applicant.find("applicant_id")                   # => Finds a single applicant
+onfido.applicant.list()                                 # => Returns all applicants
+```
+
+**Note:** Calling `onfido.applicant.destroy()` adds the applicant and all associated documents, photos, videos, checks, and reports to the deletion queue. They will be deleted 20 days after the request is made. An applicant that is scheduled for deletion can be restored but applicants that have been permanently deleted cannot.
+See https://documentation.onfido.com/#delete-applicant for more information.
+
+#### Documents
+
+Documents provide supporting evidence for Onfido checks.
+
+```java
+onfido.document.upload(inputStream, "fileName", requestBody) # => Creates a document
+onfido.document.find("document_id")                          # => Finds a document
+onfido.document.download("document_id")                      # => Downloads a document as a binary data
+onfido.document.list("applicant_id")                         # => Returns all applicant's documents
+```
+
+**Note:** The first parameter in the upload method should be an `InputStream` created from the desired file to upload.
+ 
+**Note:** The download method returns the file in the form of a `InputStream`.
+
+#### Live Photos
```

Comment:
Minor: No need to use title case for "Live Photos" here. Same as in rest of README. 

Also I'm puzzled why GitHub isn't rendering it properly in this view. The rich text looks fine...